source/connect.txt

```diff
@@ -9,5 +9,174 @@ Connect to a Deployment
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
+
+
+.. important::
+
+   Ensure that a MongoDB deployment is running before attempting to 
+   connect to it with ``mongosh``. The connection method you use 
+   depends on your deployment type.
+
+Local MongoDB Instance on Default Port
+--------------------------------------
+
+Run ``mongosh`` without any command-line options
+to connect to a MongoDB instance running on your **localhost** with 
+**default port** 27017:
+
+.. code-block:: sh
+
+   mongosh
+
+Local MongoDB Instance on a Non-Default Port
+--------------------------------------------
+
+To specify a port to connect to on localhost, use the 
+:manual:`connection string </reference/connection-string>` format. 
+
+.. example::
+
+   To connect to a MongoDB instance running on localhost with a 
+   non-default port 28015:
+
+   .. code-block:: sh
+
+      mongosh "localhost:28015"
+
+MongoDB Instance on a Remote Host
+---------------------------------
+
+To specify a remote host and port, use a 
+:manual:`connection string </reference/connection-string>`.
+
+.. example:: 
+
+   To connect to a MongoDB
+   instance running on a remote host machine on port 28015:
+
+  .. code-block:: sh
+
+     mongosh "mongodb://mongodb0.example.com:28015"
+
+.. note:: Connecting to |service|
+
+   If your remote host is a |service-fullname| cluster, you can copy 
+   your connection string from the |service| UI. To learn more, see 
+   :atlas:`Connect to a Cluster <connect-to-cluster/#use-the-connect-dialog-to-connect-to-your-cluster>`.
+
+Connection Considerations
+-------------------------
+
+Authentication
+~~~~~~~~~~~~~~
```

Comment:
I'm wondering if this should be under Considerations, or its own top-level heading like "Connect with Authentication". Thoughts?

Reply:
I think that it's logically grouped as kind of a 'connection modifier' to be applied on top of the connection string format of whatever instance type you're trying to connect to. That said, I agree that auth is more important than a 'consideration' - thinking about better names for this whole section.